app.js

```diff
@@ -61,5 +60,117 @@ function switching() {
+    technologies: ['hash', 'CSS', 'java'],
+    golive: '',
+    linkSource: '',
+  },
+
+  {
+    name: 'Multi-Post Stories 1',
+    description: "This is my best project Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever sinceLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ips",
+    featuredImage: 'images/pop-up.png',
+    technologies: ['HTML', 'CSS', 'ruby on rails'],
+    golive: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories 2',
+    description: "this is a the facebook webpage which took me ten years to build Lorem Ipsum has been the industry's standard dummy text ever sinceLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took aand i hop you enjoyed your trip down here",
+    featuredImage: 'images/pop-up.png',
+    technologies: ['python', 'react', 'ruby'],
+    golive: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories 3',
+    description: "This is my best project Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever sinceLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ips",
+    featuredImage: 'images/pop-up.png',
+    technologies: ['vue', 'Html', 'python'],
+    golive: '',
+    linkSource: '',
+  },
+  {
+    name: 'Multi-Post Stories 4',
+    description: "This is my best project Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever sinceLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ips",
+    featuredImage: 'images/pop-up.png',
+    technologies: ['hash', 'CSS', 'java'],
+    golive: '',
+    linkSource: '',
+  },
+];
+
+function fillData({
+  name, description, featuredImage, technologies,
+}) {
+  const title = document.getElementsByClassName('p-topic');
+  const descrip = document.getElementsByClassName('description');
+  const photo = document.getElementsByClassName('snapshoot');
+
+  title.innerHTML = name;
+  descrip.innerHTML = description;
+  photo.src = featuredImage;
+
+  const techs = document.querySelectorAll('skill');
+
+  //   for (const tech of techs) {
+  //     tech.innerHTML = technologies[i];
+  //   }
+  // };
```

Comment:
- [ ] Kindly note that according to the javascript [best practices for the browser](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) number 7, You are supposed to remove all commented and unwanted code from your code base as it makes it cluttered and messy. Kindly make sure that you remove them from your code base.

Reply:
DONE!